pkgs/development/interpreters/perl/default.nix

```diff
@@ -144,7 +144,7 @@ let
         substituteInPlace "$out"/lib/perl5/*/*/Config_heavy.pl \
           --replace "${libcInc}" /no-such-path \
           --replace "${
-              if stdenv.cc.cc or null != null then stdenv.cc.cc else "/no-such-path"
+              if stdenv.hasCC then stdenv.cc.cc else "/no-such-path"
```

Comment:
Why not just `optionalString` the whole `--replace` argument?

Reply:
Maybe eventually, but trying to avoid a mass rebuild for now :)